### PR TITLE
add pi gawad + christensen to whitelist

### PR DESCRIPTION
**Pull request by davidsmejia**
## Issue Number

N/A

## Purpose/Implementation Notes

Adds Gawad to the whitelist of pi's 

## Types of changes

What types of changes does your code introduce?

- New feature (non-breaking change which adds functionality)

## Functional tests

List out the functional tests you've completed to verify your changes work locally.

## Checklist

- [x] Lint and unit tests pass locally with my changes

## Screenshots

n/a

### CODE REVIEW

File: api/scpca_portal/management/commands/load_data.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 logger = get_and_configure_logger(__name__)
 s3 = boto3.client("s3", config=Config(signature_version="s3v4"))
-project_whitelist = ["murphy_chen", "green_mulcahy_levy", "dyer_chen"]
+project_whitelist = ["murphy_chen", "green_mulcahy_levy", "dyer_chen", "gawad"]
```
Comment (jaclyn-taroni) [inline api/scpca_portal/management/commands/load_data.py:22]: Christensen too?

Comment (davidsmejia) [reply api/scpca_portal/management/commands/load_data.py]: good call updated

**Review by jaclyn-taroni — APPROVED** [review]
👍

**Review by allyhawkins — APPROVED** [review]
LGTM 🚀

**Pull request merged**